bloom_nofos/nofos/nofo.py

```diff
@@ -1146,6 +1145,26 @@ def find_h7_headers(nofo):
                     }
                 )
 
+            # Check for div role="heading" aria-level="7" in subsection body
+            if subsection.body:
+                soup = BeautifulSoup(
+                    markdown.markdown(subsection.body, extensions=["extra"]),
+                    "html.parser",
+                )
+                div_h7s = soup.find_all(
+                    "div", attrs={"role": "heading", "aria-level": "7"}
+                )
+
+                for div_h7 in div_h7s:
+                    h7_headers.append(
+                        {
+                            "section": section,
+                            "subsection": subsection,
+                            "name": div_h7.get_text(strip=True),
+                            "html_id": f"{subsection.html_id}--div-h7-{len(h7_headers)}",
```

Comment:
This should just use the `subsection.html_id`. In 90%+ of cases, if people are adding an h7 manually, they aren't going to give it an id value. 

I added an `h7` manually and the link is broken 😢